ramls/request.json

```diff
@@ -141,6 +145,10 @@
         "enumeration": {
           "description": "The volume and/or issue number for the item",
           "type": "string"
+        },
+        "effectiveCallNumberComponents": {
+          "description": "Elements of a full call number generated from the item or holding",
```

Comment:
Within the circulation context, I don't think it is relevant that the elements are generated / derived from the item or holdings.